### ISSUE TITLE
Hashrate Halved after new hardfork update?

### ISSUE THREAD

**Issue by TheMitchilling** [fireice-uk/xmr-stak#2032]
hi,
not sure if this is an issue but after the new hardfork, the hashrate of my rig has halved is this normal? from 800 to 400 hs. is there a setting i may not know about ?

**Comment by TheMitchilling** [comment]
both GTX 760 and AMD hd6970 dropped by about half. cpu hs remain unaffected

**Comment by Spudz76** [comment]
Yes, normal, new algo is tougher on older GPUs.  Some CPUs are actually faster with the new algo, or unchanged.

**Comment by TheMitchilling** [comment]
Thank you i thought this was the case any Gpus that you would recommend for the new algorithm?

**Comment by Spudz76** [comment]
Algo too new to know what's the best yet.  But probably still Vega56 with as little memory as you can get.  GTX970 was not hurt much.

We only know how well it works when users test it (and report back somehow)
